exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,33 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if(num<0){
+    return true;
+  }else if(num>0){
+    return false;
+  }
+}
+
+function isBetween5and10(num){
+  if(num>=5 && num<=10){
+    return true;
+  }else{
+    return false;
+  }
```

Comment:
Great job Cuneyt. Mine is a little bit different, I didn't use If conditions. 
Above your code:
  if(num>=5 && num<=10){
    return true;
  }else{
    return false;
  }

Mine:
function isBetween5and10(y) {
  return y >= 5 && y <= 10;
}

Reply:
Great feedback so far Ekrem, thank you for contributing to the review 👍 

If you'd like to demonstrate code in a github review, all you need to do is use 3x backticks, if you want to tell github which language to format for you can add that as well. I'll share a screenshot below to as an example:

**Syntax Example**
![syntaxexample](https://user-images.githubusercontent.com/2116227/190838661-53e3ccd8-afad-430b-a94b-238640451318.JPG)

**How this syntax will display in the comment**
![syntaxpreview](https://user-images.githubusercontent.com/2116227/190838674-34fd75a9-8c00-4400-a726-c74def9a6e52.JPG)